README.md

```diff
@@ -218,6 +218,33 @@ git-xargs --github-org my-github-org \
   "$(pwd)/scripts/my-ruby-script.rb"
 ```
 
+## Debugging runtime errors
+
+By default, `git-xargs` will conceal runtime errors as they occur because its log level setting is `INFO` if not overridden by the `--loglevel` flag.
+
+To see all errors your script or command may be generating, be sure to pass `--loglevel debug` when running your `git-xargs` command, like so:
```

Comment:
Capitalized `INFO` versus lower case `debug`. Which should we use?

Reply:
Thanks - fixed in cb00308